Acrolinx.Sidebar/AcrolinxSidebar.cs

```diff
@@ -395,6 +395,12 @@ public String Check(IDocument document)
             string checkId = check.checkId;
             return checkId;
         }
+        public void CancelCheck()
+        {
+            var code = "new function(){ return acrolinxSidebar.onGlobalCheckRejected(); }";
+            Eval(code);
+            Logger.AcroLog.Info("Check has been cancelled due to some errors in document.");
```

Comment:
"due to some errors in document" how do you know?

Reply:
log message improved.